src/Compilers/CSharp/Portable/Generated/BoundNodes.xml.Generated.cs

```diff
@@ -11335,6 +11335,1816 @@ public override BoundNode VisitExpressionWithNullability(BoundExpressionWithNull
         }
     }
 
+    internal sealed partial class NullabilityRewriter : BoundTreeRewriter
+    {
+        private readonly ImmutableDictionary<BoundExpression, TypeSymbolWithAnnotations> _topLevelNullabilities;
```

Comment:
>_topLevelNullabilities [](start = 89, length = 22)

If the dictionary is storing top-level and nested nullability, should it be renamed?